Split-Directions.parset

```diff
@@ -182,21 +183,26 @@ phaseup_concat.argument.msout.overwrite        = True
 phaseup_concat.argument.steps                  = [count]
 phaseup_concat.argument.numthreads             = {{ max_dppp_threads }}
 
+# generate mapfile of the output file
+createmap_selfcal.control.kind             = plugin
+createmap_selfcal.control.type             = createMapfile
+createmap_selfcal.control.method           = mapfile_from_folder
+createmap_selfcal.control.mapfile_dir      = input.output.mapfile_dir
+createmap_selfcal.control.filename         = createmap_selfcal.mapfile
+createmap_selfcal.control.folder           = {{ job_directory }}/delay_solve
```

Comment:
this should be {{ job_directory }}/../Delay-Calibration/delay_solve 

but check Delay-Calibration for how it's done